modules/gapi/doc/slides/get_sty.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+MTHEME_VER=2fa6084b9d34fec9d2d5470eb9a17d0bf712b6c8
```

Comment:
Enable errors in the beginning of the script: `set -e`

Reply:
done